generator/internal/language/rust.go

```diff
@@ -776,97 +780,117 @@ func (c *rustCodec) formatDocComments(documentation string, state *api.APIState)
 
 	links := map[string]bool{}
 	var results []string
-	for _, line := range strings.Split(documentation, "\n") {
-		if inBlockquote {
-			switch {
-			case line == "```":
-				inBlockquote = false
-				results = append(results, "```")
-			case strings.HasPrefix(line, blockquotePrefix):
-				results = append(results, strings.TrimPrefix(line, blockquotePrefix))
-			default:
-				inBlockquote = false
+	links := map[string]bool{}
```

Comment:
dupped definition of `links`?

Reply:
Thanks for pointing this out - looks like this was side effect of the rebase. Cleaned it up.